hw_3_zd_2.js

```diff
@@ -0,0 +1,18 @@
+
+function printJobName({
+    name:nameUser,
+    job:{
+        name,
```

Comment:
this is not correct way of desctructurization. Please return my code and make changes ONLY in input parameters of the function

Reply:
Done